src/Adjudications/AdjudicationHistory/AdjudicationHistory.style.js

```diff
@@ -0,0 +1,38 @@
+import styled from 'styled-components'
+import Select from '@govuk-react/select'
+import GridCol from '@govuk-react/grid-col'
+import { H3 } from '@govuk-react/heading'
+import { GREY_3, WHITE } from 'govuk-colours'
+import { SPACING, FONT_SIZE, MEDIA_QUERIES } from '@govuk-react/constants'
+
+export const FullWidthSelect = styled(Select)`
+  select {
+    width: 100% !important;
+    background-color: ${WHITE};
+  }
+`
+
+export const SearchArea = styled.div`
+  background: ${GREY_3};
+  padding: ${SPACING.SCALE_5};
+  margin-bottom: ${SPACING.SCALE_3};
+`
+
+export const LargeSreenOnlyGridCol = styled(GridCol)`
+  display: none;
+  ${MEDIA_QUERIES.TABLET} {
+    display: block;
+    height: 100%;
+  }
+`
+export const FiltersLabel = styled(H3)`
+  padding-left: ${SPACING.SCALE_2};
+  margin-bottom: ${SPACING.SCALE_2};
+  margin-top: -${SPACING.SCALE_2};
+`
+export const DateRangeLabel = styled.span`
+  font-weight: bold;
+  padding-left: ${SPACING.SCALE_2};
+  font-size: ${FONT_SIZE.SIZE_18}
```

Comment:
Any reason why it's 18?  I'd also recommend using `${typography.font({ size: 19 })};` from `@govuk-react/lib`.  Gives you responsive text size and 19 is generally the body default.

Reply:
I didn't know this - I was trying to use something "standard" and there wasn't a constant for 19. I've fixed this.